aii-ks/src/main/perl/ks.pm

```diff
@@ -610,7 +621,7 @@ sub ksuserscript
 
     print <<EOS;
 pushd /root
-wget --output-document=userscript $url
+wget --timeout 60 --output-document=userscript $url
```

Comment:
i understand that you want to protect against hanging, but if wget times out, should we abort rather than continue?

Reply:
That's a good question. In our case, if `userscript` does not run for whatever reason, then the build will fail pretty soon after that anyway.

Reply:
I've added `|| fail ...`